.github/workflows/ci.yml

```diff
@@ -30,23 +30,24 @@ jobs:
           export TEST_PHP_ARGS="-n -d extension=mbstring.so -d extension=modules/mailparse.so"
           php run-tests.php -P --show-diff tests
   windows:
-    defaults:
-      run:
-        shell: cmd
+    runs-on: windows-latest
+    continue-on-error: false
     strategy:
+      fail-fast: false
       matrix:
-          version: ["7.4", "8.0"]
+          version: ["7.4", "8.0", "8.1", "8.2", "8.3"]
```

Comment:
[PHP < 8.0.0 will no longer successfully built on windows-latest](https://github.com/php/setup-php-sdk/pull/1) (currently windows-2022). Using windows-2019 instead should be fine for the time being.